Antidote/ActiveCallViewController.m

```diff
@@ -126,14 +131,19 @@ - (void)createMuteButton
     [self.controlsContainerView addSubview:self.speakerButton];
 }
 
-- (void)createPauseButton
+- (void)createResumeButton
 {
-    self.pauseButton = [self createButtonWithImageName:@"call-play" action:@selector(pauseButtonPressed)];
+    if (self.resumeButton) {
+        return;
+    }
```

Comment:
I think there is no need for this check as button is created only in `viewDidLoad`